src/api.v2/helpers/s3/patchCellSetsObject.js

```diff
@@ -0,0 +1,39 @@
+const jsonMerger = require('json-merger');
+
+const bucketNames = require('./bucketNames');
+const getObject = require('./getObject');
+const putObject = require('./putObject');
+
+const validateRequest = require('../../../utils/schema-validator');
+
+const patchCellSetsObject = async (experimentId, patch) => {
+  const initialCellSet = await getObject({
```

Comment:
Maybe `prePatchCellSet`, initial sounds more like it would be a default or something